src/nft/components/profile/list/PriceTextInput.tsx

```diff
@@ -42,25 +43,28 @@ const GlobalPriceIcon = styled.div`
   background-color: ${({ theme }) => theme.backgroundSurface};
 `
 
-const WarningMessage = styled(Row)<{ warningType: WarningType }>`
+const WarningRow = styled(Row)`
+  gap: 4px;
+`
+
+const WarningMessage = styled(Row)<{ color: string }>`
```

Comment:
the styled-components naming convention for naming a prop that already exists is by appending a `$` so `color` should be `$color`